services-examples/src/main/java/io/scalecube/services/examples/BootstrapExample.java

```diff
@@ -34,21 +33,10 @@ public static void main(String[] args) throws Exception {
         Microservices.builder()
             .discovery(ScalecubeServiceDiscovery::new)
             .transport(ServiceTransports::rsocketServiceTransport)
-            .gateway(
-                GatewayConfig.builder("http", HttpGatewayStub.class)
-                    .port(8181)
-                    .build()) // override default port
-            .gateway(
-                GatewayConfig.builder("ws", WebsocketGatewayStub.class)
-                    .port(9191)
-                    .addOption(
-                        WebsocketGatewayStub.WS_SPECIFIC_OPTION_NAME,
-                        "500") // override default value of specific option
-                    .build())
+            .gateway(opts -> new HttpGatewayStub(opts.id("http").port(8181)))
```

Comment:
lets find better name then stub

Reply:
Changed name to end with postfix -- _Example_ instead of _Stub_